api/islandora_xacml_api.install

```diff
@@ -173,10 +173,17 @@ EOQ;
   $ds = $parts[1];
   $object_to_update = islandora_object_load($pid);
 
+  if (!isset($object_to_update[$ds])) {
+    $temp_ds = $object_to_update->constructDatastream($ds);
+  }
+  else {
+    $temp_ds = $object_to_update[$ds];
+  }
+
   $manageable_by_user = 'isManageableByUser';
   $manageable_by_role = 'isManageableByRole';
-  $object_to_update[$ds]->relationships->remove(ISLANDORA_RELS_EXT_URI, $manageable_by_user);
-  $object_to_update[$ds]->relationships->remove(ISLANDORA_RELS_EXT_URI, $manageable_by_role);
+  $temp_ds->relationships->remove(ISLANDORA_RELS_INT_URI, $manageable_by_user);
```

Comment:
I'm pretty sure this should just be removing RELS-EXT NS as that is what has been queried for https://github.com/Islandora/islandora_xacml_editor/pull/115/files#diff-c2819f5529321c1300b2e202ca30a4daR131